libobs-d3d11/d3d11-subsystem.cpp

```diff
@@ -454,6 +454,32 @@ try {
 	return false;
 }
 
+static inline bool IsNvidiaDriver527OrNewer(uint64_t version)
+{
+	const uint16_t aa = (version >> 48) & 0xffff;
+	const uint16_t bb = (version >> 32) & 0xffff;
+	const uint16_t ccccc = (version >> 16) & 0xffff;
+	const uint16_t ddddd = version & 0xffff;
+
+	// 31.0.15.2737
+	if (aa >= 31 && bb >= 0 && ccccc >= 15 && ddddd >= 2737) {
+		return true;
+	} else {
+		return false;
+	}
+}
+
+bool gs_device::CheckFastClear()
+{
+	if (VendorId == 0x10de && IsNvidiaDriver527OrNewer(DriverVersion)) {
+		return true;
+	} else if (VendorId == 0x10de) {
+		return false;
+	} else {
+		return true;
+	}
```

Comment:
Could be shortened to `return VendorId != 0x10de || IsNvidiaDriver527OrNewer(DriverVersion)`

Reply:
That seems fine. Can change later.

Reply:
Used your implementation suggested in the other review thread.